### PR TITLE
#1926 added DetailDisclosureButton for iOS native apps, JApp annotation

### PR DESCRIPTION
**Pull request by alexarm**

### CODE REVIEW

**Comment by mf-code** [comment]
[5454] - AFp8QSQ2

**Comment by mf-code** [comment]
[5454] - I5agHmJy

**Comment by mf-code** [comment]
[5454] - Allure report on Netlify: https://5ed0cffc1c63910704cdda9f--musing-swartz-6fc060.netlify.app

File: jdi-light-mobile-tests/src/test/java/nativeapp_ios/CalendarAppTests.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package nativeapp_ios;
+
+import nativeapp.ios.calendar.*;
```
Comment (codacy-review) [inline jdi-light-mobile-tests/src/test/java/nativeapp_ios/CalendarAppTests.java:3]: Codacy Issue found: [Avoid unused imports such as 'nativeapp.ios.calendar'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5606286)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 17
- Added 1
           

Complexity increasing per file
==============================
- jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/elements/common/ios/app/DetailsDisclosureButton.java  1
- jdi-light-mobile-tests/src/test/java/nativeapp_ios/CalendarAppTests.java  2
- jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/settings/MobileSettings.java  2
- jdi-light-mobile-tests/src/test/java/nativeapp_ios/CalendarAppTestsInit.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5606286&bid=18367162)

**Comment by mf-code** [comment]
[5455] - B0WXcCLS

**Comment by mf-code** [comment]
[5455] - 59onQ8m7

**Comment by mf-code** [comment]
[5455] - Allure report on Netlify: https://5ed0d8c0a6539ac1c343578d--musing-swartz-6fc060.netlify.app

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 17
- Added 1
           

Complexity increasing per file
==============================
- jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/elements/common/ios/app/DetailsDisclosureButton.java  1
- jdi-light-mobile-tests/src/test/java/nativeapp_ios/CalendarAppTestsInit.java  1
- jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/settings/MobileSettings.java  2
- jdi-light-mobile-tests/src/test/java/nativeapp_ios/CalendarAppTests.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5606286&bid=18367162)

File: jdi-light-mobile-tests/src/test/java/nativeapp_ios/CalendarAppTests.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
```
Comment (codacy-review) [inline jdi-light-mobile-tests/src/test/java/nativeapp_ios/CalendarAppTests.java:3]: Codacy Issue found: [Avoid unused imports such as 'nativeapp.ios.calendar'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5606286)

**Comment by mf-code** [comment]
[5457] - MkzokCZM

**Comment by mf-code** [comment]
[5457] - iJpOI9PY

**Comment by mf-code** [comment]
[5457] - Allure report on Netlify: https://5ed0f1531c6391c18dcddc75--musing-swartz-6fc060.netlify.app

**Pull request merged**